core/src/com/unciv/logic/MapSaver.kt

```diff
@@ -10,20 +10,32 @@ object MapSaver {
     fun json() = GameSaver.json()
 
     private const val mapsFolder = "maps"
+    private const val saveZipped = false
 
     private fun getMap(mapName:String) = Gdx.files.local("$mapsFolder/$mapName")
 
+    fun mapFromSavedString(mapString: String): TileMap {
+        val unzippedJson = try {
+            Gzip.unzip(mapString)
+        } catch (ex: Exception) {
+            mapString
```

Comment:
U wot m8

Reply:
For tests it is easier to get readable json out of the clipboard save feature, but I left the `private const val saveZipped = false` switch on, not planned. Heck, once the Options screen is properly redesigned we could pack that into options, godmode, viewentiremap, whatnot - maybe a tab that requires a secret handshake.

That is the counterpart - you can feed the loader gzipped json or plain json, it will eat both.